src/toil_scripts/batch_alignment/bwa_alignment.py

```diff
@@ -226,40 +257,43 @@ def main():
     # Run subparser
     parser_run = subparsers.add_parser('run', help='Runs the BWA alignment pipeline')
     group = parser_run.add_mutually_exclusive_group(required=True)
-    parser_run.add_argument('--config', default='bwa-alignment.config', type=str,
+    parser_run.add_argument('--config', default='config-toil-bwa.yaml', type=str,
                             help='Path to the (filled in) config file, generated with "generate-config".')
-    group.add_argument('--manifest', default='bwa-alignment-manifest.csv', type=str,
+    group.add_argument('--manifest', default='manifest-toil-bwa.tsv', type=str,
                        help='Path to the (filled in) manifest file, generated with "generate-manifest". '
                             '\nDefault value: "%(default)s".')
-    group.add_argument('--sample', default=None, nargs='3', type=str,
-                       help='Space delimited sample UUID and paired fastq files in the format: uuid url1 url2')
-    parser_run.add_argument('--output-dir', default=None, help='Full path to directory or filename where '
-                                                               'final results will be output')    
-    parser_run.add_argument('-s', '--suffix', default='.bqsr', help='Additional suffix to add to the names of the output files.')
+    group.add_argument('--sample', nargs='+', action=required_length(2, 3),
+                       help='Space delimited sample UUID and fastq files in the format: uuid url1 [url2].')
+    # Print docstring help if no arguments provided
+    print sys.argv
```

Comment:
I would remove this.

If you were going to keep it, I think @hannes-ucsc would want `repr(sys.argv)`. I would also want this to be `log.info`/`log.debug`ed, not `print`ed.